webhook/go.mod

```diff
@@ -27,3 +27,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/CycloneDX/cyclonedx-go => github.com/CycloneDX/cyclonedx-go v0.6.0
```

Comment:
what does this do? Looks like a no-op?

https://github.com/vmware-tanzu/cartographer-conventions/pull/267/files#diff-bf6e2cea0f35adacfeb048cf4e8d84dc5efcf723f2e14053c81e29331e331a29R6